### DISCUSSION TITLE
Database Enumerations: Take Me To Your Leader

### DISCUSSION THREAD

**Discussion by textbrowser** [textbrowser/biblioteq#219] [Show and tell]
Select a section and BQ will scroll to it. Smooth.

![Screenshot_20220721_070546](https://user-images.githubusercontent.com/10701156/180199460-563b8b75-07ae-4492-81ad-076ed668ac3e.png)

**Comment by textbrowser** [comment]
This means that BQ catalogs itself.

**Reply by meteos77** [reply]
glad to see you spending time on BiblioteQ again :-)

I just saw the new feature, nice improvement

suggestion if it's easy / development time : to have the maximum of lines per section for example for the number of borrowing days, not to have 4 lines but the 7 lines

or in the display area of the section have only the selected section

**Reply by meteos77** [reply]
for my understanding, can you please change the wording : "This means that BQ catalogs itself."

**Comment by textbrowser** [comment]
When you click an item, it takes you to the section. That's an index. So, BQ catalogs itself. And you can download BQ and save it as a grey literature item. Two ways BQ catalogs itself. That's dope.
